wlc/test_wlc.py

```diff
@@ -135,6 +135,30 @@ def test_repr(self):
         self.assertIn("'slug': 'hello'", repr(obj))
         self.assertIn("'slug': 'hello'", str(obj))
 
+    def test_create_project(self):
+        resp = Weblate().create_project(
+            "Hello", "hello", "http://example.com/", "Malayalam", "ml"
+        )
+        self.assertIn("'name': 'Hello', 'slug': 'hello'", str(resp))
+
+    def test_create_component(self):
+        resp = Weblate().create_component(
+            project="hello",
+            branch="master",
+            file_format="po",
+            filemask="po/*.po",
+            git_export="",
+            license="",
+            license_url="",
+            name="Weblate",
+            slug="weblate",
+            repo="file:///home/nijel/work/weblate-hello",
+            template="",
+            new_base="",
+            vcs="git",
+        )
+        self.assertIn("'name': 'Weblate', 'slug': 'weblate'", str(resp))
```

Comment:
Why aren't we parsing the response and testing specific component attributes?

Reply:
I'll update the tests. 😅